File: CH1-input-process-output/EX7-miles-per-gallon/milesPerGallon.cpp
#include <iostream>
using namespace std;

int main()
{
    int MPG;
    int milesDriven;
    int gallons;

    // Input
    cout << "Miles Driven :";
    cin >> milesDriven;
    cout << "Gallons Gas  :";
    cin >> gallons;

    // Process
    MPG = milesDriven / gallons;

    // Output
    cout << "Miles-per-Gallon : " << MPG << endl;

    return 0;
}

File: CH1-input-process-output/EX7-miles-per-gallon/milesPerGallon.php
<?php
// Input
$milesDriven = readline("Miles Driven :");
$gallons     = readline("Gallons Gas :");

// Process
$MPG = $milesDriven / $gallons;
$MPG = number_format((float) $MPG, 1, '.', '');

// Output
echo "Miles-per-Gallon :" . $MPG;
